common/modules/olm/manifests/1.4.0/amq-streams.v1.4.0.clusterserviceversion.yaml

```diff
@@ -397,7 +397,7 @@ spec:
   - name: Product Page
     url: https://access.redhat.com/products/red-hat-amq
   - name: Documentation
-    url: https://access.redhat.com/documentation/en-us/red_hat_amq/7.5/html/using_amq_streams_on_openshift_container_platform/index
+    url: https://access.redhat.com/documentation/en-us/red_hat_amq/7.6/html/using_amq_streams_on_openshift_container_platform/index
```

Comment:
https://access.redhat.com/documentation/en-us/red_hat_amq/7.6/html-single/using_amq_streams_on_openshift/index

Reply:
It seems you are getting old files :-)